tests/test_jira.py

```diff
@@ -16,7 +16,7 @@ def json(self):
             return self.json_data
 
     # Careful, args needs to be a tuple, and that always ends with a "," character in Python!!
-    if args == ('mock-ucsc-cgl/search?jql=id=REAL-ISSUE-1',):
+    if args == ('mock-ORG/search?jql=id=REAL-ISSUE-1',):
```

Comment:
I would use a more URL-like string here, i.e., starting with `https://mock....`, similar to how I've done it in `test_github_issue`.

Reply:
changed to 'https://mock-org.atlassian.net/'